# Goal
string object_name
---
# Result
bool success
string message
---
# Feedback
float32 percentage_complete
string status_message
